letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -1344,7 +1344,8 @@ def get_index_base():
 
 
 def main():
-    pip_version = StrictVersion(check_output(['pip', '--version'])
+    executable = python_executable or 'python'
```

Comment:
The meaning might be clearer if we shallow-import `executable` and call the result "python", making this `python = sys.executable or 'python'`. Idiomatic Python readers familiar with the stdlib will readily identify sys.executable, and `python` will stand in for ".../python", its usual value.

Reply:
Good idea. I did that.